src/contract/mod.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2019 Centrality Investments Limited
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub mod data_table;
+use bit_reverse::ParallelReverse;
```

Comment:
Would prefer to keep the imports sorted here. Also does `mod data_table` need to be pub?
```
mod data_table;

use alloc::vec::Vec;
use bit_reverse::ParallelReverse;
pub use data_table::DataTable;
```

Reply:
you are right, `mod data_table` doesn't need to be pub